web/react/components/command_list.jsx

```diff
@@ -44,6 +46,30 @@ export default class CommandList extends React.Component {
             this.props.channelId,
             cmd,
             true,
+            false,
+            function success(data) {
```

Comment:
You'll may ask yourself why am I triying to contribute, once again I'm not familiar with es6 (and react) for now…

Can you give me some example on this too ?

Again, I just copied `getSuggestedCommands` here, and added a boolean.